pkg/util/flowcontrol/throttle_test.go

```diff
@@ -59,16 +59,16 @@ func TestMultithreadedThrottling(t *testing.T) {
 	r.Accept()
 	// start the thundering herd
 	close(startCh)
-	// wait for the first signal that we collected 100 tokens
+	// wait for the first signal that we collected 101 tokens
 	<-endCh
 	// record wall time
 	endTime := time.Now()
 
 	if duration := endTime.Sub(startTime); duration < time.Second {
-		// We shouldn't be able to get 100 tokens out of the bucket in less than 1 second of wall clock time, no matter what
-		t.Errorf("Expected it to take at least 1 second to get 100 tokens, took %v", duration)
+		// We shouldn't be able to get 101 tokens out of the bucket in less than 1 second of wall clock time, no matter what
```

Comment:
that's my best guess. I'd rather explicitly tolerate a 1% drift with something like `duration < (time.Second * 99 / 100)`